server/@types/express/index.d.ts

```diff
@@ -1,9 +1,8 @@
 import type { PageFlow, PrisonerSummary } from 'viewModels'
-import type { ArchiveGoalForm, CreateGoalsForm, UpdateGoalForm } from 'forms'
+import type { ArchiveGoalForm, CreateGoalsForm, HighestLevelOfEducationForm, UpdateGoalForm } from 'forms'
 import type {
   AdditionalTrainingForm,
   AffectAbilityToWorkForm,
-  HighestLevelOfEducationForm,
```

Comment:
I've moved `HighestLevelOfEducationForm` from the `inductionForms` module to the `forms` module (see next file below 👇 )